test/integration/scheduler/util.go

```diff
@@ -72,19 +72,22 @@ type testContext struct {
 	cancelFn        context.CancelFunc
 }
 
-func createAlgorithmSourceFromPolicy(policy *schedulerapi.Policy, clientSet clientset.Interface) schedulerconfig.SchedulerAlgorithmSource {
-	policyString := runtime.EncodeOrDie(latestschedulerapi.Codec, policy)
+func createAlgorithmSourceFromPolicy(policy *schedulerapi.Policy, clientSet clientset.Interface) schedulerapi.SchedulerAlgorithmSource {
+	// Serialize the Policy object into a ConfigMap later.
+	info := scheme.Codecs.SupportedMediaTypes()[0]
```

Comment:
what is this assuming is at index 0 of SupportedMediaTypes()? yaml? json?

Reply:
By default index 0 is JSON, but we don't quite care about which format it's going to be stored into the configmap. We will parse that {JSON/YAML} (in the configmap) to an internal Policy obj.

Reply:
prefer something like this:
```
info, ok := runtime.SerializerInfoForMediaType(scheme.Codecs.SupportedMediaTypes(), runtime.ContentTypeJSON)
if !ok {
  panic("could not find json serializer")
}
```